mozci/push.py

```diff
@@ -1076,6 +1076,13 @@ def get_regressions(
             if not prior_regression and count <= MAX_DEPTH:
                 regressions[name] = int(count) if count > 0 else 0
 
+            # if confirmed = True; then assume regression
+            if isinstance(failure_summary, GroupSummary):
+                if failure_summary.is_confirmed_failure():
+                    count = 0
+                elif failure_summary.is_confirmed_failure() is False:
+                    count += 1
```

Comment:
Maybe:
```suggestion
                    count *= 2
```
or even `count = MAX_DEPTH + 1` so it is removed from the list of regressions?